### ISSUE TITLE
Unable to catch CouldNotRetrieveTranscript exception

### ISSUE THREAD

**Issue by paramt** [jdepoix/youtube-transcript-api#17]
As far as I can tell, there is no way to cleanly handle an exception from the `get_transcript` method.

```python
from youtube_transcript_api import YouTubeTranscriptApi as ytcc

try:
    captions = ytcc.get_transcript("MHTizZ_XcUM", languages=["fr"])
except:
    captions = ytcc.get_transcript("MHTizZ_XcUM", languages=["en"])
```
This code shouldn't work at first because the video does not have French captions, so I would like to handle it by getting the English captions. But, this is what is printed out instead:

```
Could not get the transcript for the video https://www.youtube.com/watch?v=MHTizZ_XcUM! This usually happens if one of the following things is the case:
 - subtitles have been disabled by the uploader
 - none of the language codes you provided are valid
 - none of the languages you provided are supported by the video
 - the video is no longer available.

If none of these things is the case, please create an issue at https://github.com/jdepoix/youtube-transcript-api/issues
```

Is this intentional or am I missing something? I would like to be able to handle the exception without anything being printed out.

**Comment by jdepoix** [comment]
I am pretty sure that your code is working as intended. If you print out `captions` you'll probably see that it contains the subtitles you want. The message you are seeing is just a log message, which is logged by the API using [the `logging` package](https://docs.python.org/3/library/logging.html). If you don't have configured where you want your log messages to go, it'll just be printed to stdout

However, I do agree that this message should be logged with a different loglevel than "error", as it shouldn't be considered an error in cases such as yours. I will change this tomorrow.

**Comment by jdepoix** [comment]
@paramt BTW, what you are trying to achieve with the code you posted, can be done much easier: 
```python
from youtube_transcript_api import YouTubeTranscriptApi

captions = YouTubeTranscriptApi.get_transcript("MHTizZ_XcUM", languages=["fr", "en"])
`
``
This will fallback on `en` if it fails to retrieve the subtitles for `fr`.

**Comment by jdepoix** [comment]
The logging has been removed in #18 and released in v0.1.6.
